dicomImport/matRad_importDicomSteeringPhotons.m

```diff
@@ -0,0 +1,103 @@
+function [stf, pln] = matRad_importDicomSteeringPhotons(pln)
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% matRad function to import a matRad stf struct from dicom RTPLAN data
+% 
+% call
+%   [stf, pln] = matRad_importDicomSteeringPhotons(pln)
+%
+% input
+%   pln:            matRad pln struct with meta information (collimation
+%                   data included)
+%
+% output
+%   stf             matRad stf struct
+%   pln             matRad pln struct
+%
+% References
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% 
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2015 the matRad development team. 
+% 
+% This file is part of the matRad project. It is subject to the license 
+% terms in the LICENSE file found in the top-level directory of this 
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part 
+% of the matRad project, including this file, may be copied, modified, 
+% propagated, or distributed except according to the terms contained in the 
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% get fields possessing a field weight vector greater than 0
+Fields = pln.Collimation.Fields([pln.Collimation.Fields(:).Weight] > 0);
+
+[UniqueComb,ia,ib] = unique( vertcat([Fields(:).GantryAngle], [Fields(:).CouchAngle])','rows');
+
+% return corret angles to pln, because some angle derivations might be 
+% only in the control point sequences
+pln.gantryAngles = UniqueComb(:,1)';
+pln.couchAngles = UniqueComb(:,2)';
+
+stf = struct;
+% loop over all fields
+for i = 1:size(UniqueComb,1)
+    % set necessary steering information 
+    stf(i).gantryAngle  = UniqueComb(i,1);
+    stf(i).couchAngle   = UniqueComb(i,2);
+    stf(i).isoCenter    = pln.isoCenter;
+    
```

Comment:
Currently we assume a constant isocenter, but I could change this in the import with relatively little effort. We already have a function in the plan import that checks if the isocenter is consistent. [RTPlan_import](https://github.com/markbangert/matRad/blob/8a060b333a400bdaaafe8452de6e5026509550c2/dicomImport/matRad_importDicomRTPlan.m#L89)